### PR TITLE
added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Saad-Abbasi**
## Description

This pull request adds the missing CONTRIBUTING.md file to the prisma-express-typescript-boilerplate repository. The CONTRIBUTING.md file provides guidelines and instructions for contributors, making it easier for them to understand the contribution process and get involved in the project.

## Motivation

The absence of a CONTRIBUTING.md file in the repository can sometimes create confusion for new contributors who are unfamiliar with the project's conventions and guidelines. By adding this file, we can provide clear instructions on how to contribute, including steps to set up the project, guidelines for code style and formatting, and any specific requirements for submitting pull requests.

## Changes Made

- Created CONTRIBUTING.md file with guidelines for contributors.
- Included instructions for setting up the project, installing dependencies, and running tests.
- Added guidelines for code style, commit message conventions, and pull request submission.
- Mentioned the project's Code of Conduct to ensure a respectful and inclusive environment for all contributors.

## Related Issues

This pull request does not address any specific issue.

## Future Improvements

In the future, we can consider expanding the CONTRIBUTING.md file to include more detailed information, such as examples of code contributions, documentation guidelines, and information on how to run the project locally for development.

## Additional Notes

I'm open to feedback and suggestions from the maintainers and the community. I'm excited to contribute to the prisma-express-typescript-boilerplate project and help improve the onboarding experience for new contributors.

### CODE REVIEW

**Comment by antonio-lazaro** [comment]
Thank you @Saad-Abbasi for your contribution! I think there was an unintended addition of the `package.json` file to the commit, but the `CONTRIBUTING.md` file is fine. Please, remove the `package.json` file in order to merge it.

### Future Improvement Comments
In the future, I think that we could add more information to the contributing file, however, in order not to repeat information, as in the case of the `information on how to run the project locally for development`, I would reference the file where they can find this info (e. g. `README.md`).

Thank you!

File: package.json
Comment (antonio-lazaro) [inline package.json:1]: I think this is an unintended addition.

**Review by antonio-lazaro — CHANGES_REQUESTED** [review]
Remove `package.json` changes.

File: package.json
Comment (Saad-Abbasi) [reply package.json:1]: Yes, you are right, that was something wrong commented. thanks for reviewing. 
I have fixed it.

**Review by antonio-lazaro — APPROVED** [review]
Perfect. Thank you!

**Pull request merged**